src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/MacApplicationBuilder.java

```diff
@@ -123,6 +126,18 @@ private static void validateAppVersion(Application app) throws ConfigException {
         }
     }
 
+    private static void validateAppContentDirs(Application app) {
+        for (var contentDir : app.contentDirs()) {
+            if (!CONTENTS_SUB_DIRS.stream()
+                    .anyMatch(subDir -> contentDir.getFileName().toString()
+                                                  .equalsIgnoreCase(subDir))) {
+                Log.info(MessageFormat.format(I18N.getString(
```

Comment:
Can be simplified down to:
```
Log.info(I18N.format("warning.non.standard.contents.sub.dir", contentDir));
```

Reply:
Fixed. I copy-paste this code. We have many places with `MessageFormat.format(I18N.getString())`. Should I file follow up bug to clean-up it?